### ISSUE TITLE
Plugin does not work with Calibre 6.4 and BluFire-Reader epub's

### ISSUE THREAD

**Issue by kurtigoedel** [noDRM/DeDRM_tools#152]
### Question / bug report

I used to be able to convert epub ebooks I downloaded using the BlueFire reader app (Android) into Calibre using the plugin.
This no longer works, Calibre reports that the epub book is still DRM protected when trying to convert.
This is probably due to a newer version of the BlueFire reader, because I now have to enter my Adobe ID and password every time I download an eBook to the BlueFire reader, which was not the case before.

Translated with www.DeepL.com/Translator (free version)

### Which version of Calibre are you running?

6.4

### Which version of the DeDRM plugin are you running?

10.0.3

### Log output

```log
Paste log output here.
```

**Comment by ElleKayEm** [comment]
Yeah, it seems like DeDRM would not be able to retrieve your Adobe Key in that case.  I suggest looking at the DeACSM plugin which allows you to add the ACSM to calibre and download the epub or pdf directly into calibre: https://www.mobileread.com/forums/showthread.php?t=341975
